tests/installation/finish_desktop.pm

```diff
@@ -22,13 +22,7 @@ sub run {
 
     # live may take ages to boot
     my $timeout = 600;
-    assert_screen "generic-desktop", $timeout;
-
-    ## duplicated from second stage, combine!
-    if (check_var('DESKTOP', 'kde')) {
-        send_key "esc";
-        assert_screen "generic-desktop", 25;
-    }
+    assert_screen([qw(generic-desktop opensuse-welcome)], $timeout);
```

Comment:
could you please only add the tag "opensuse-welcome" when we would expect the opensuse-welcome package to be installed? E.g. `is_opensuse && !is_leap('<15.2') && !is_live_cd` or something? If there isn't a `is_opensuse_welcome_applicable` maybe it's time to add it now :)

Reply:
there is `opensuse_welcome_applicable`, also see #8399